lib/agent/socket_util.js

```diff
@@ -120,7 +115,7 @@ exports.secureSocket = function (socket, host, agent, mock) {
 function isOcspValidationDisabled(host) {
   // ocsp is disabled if insecure-connect is enabled, or if we've disabled ocsp
   // for non-snowflake endpoints and the host is a non-snowflake endpoint
-  return GlobalConfig.isInsecureConnect() ||
+  return GlobalConfig.isOCSPChecksDisabled() ||
```

Comment:
Does this cover also a case when someone sets `insecureMode`?

Reply:
Suppose you mean ` insecureConnect`, Yes. In the core.js, the driver will take both insecureConnect and disableOCSPchecks. The disableOCSPChecks option is prioritized.